docs/walkthroughs/access-mobile-client-build-logs.adoc

```diff
@@ -3,57 +3,46 @@
 
-image::images/mobile-ci-cd-access-logs-0.png[access-build-from-notification]
+The latest build, as well as previous builds, has a `View Log` link associated with them to access the logs of that build.
 
-Otherwise, navigate through the menu
-* Expand the side-bar menu through the hamburger icon
-* Hover over `Builds`
-* Click `Pipelines`
+image::images/mobile-ci-cd-access-logs-0.png[access-build-from-notification][align="center"]
 
-image::images/mobile-ci-cd-access-logs-1.png[access-build-from-menu]
-
-There you should be able to find a `View Log` link.
-
-image::images/mobile-ci-cd-access-logs-2.png[pipeline-build]
 
 ==== Logging into Jenkins to access the logs
 
-After you clicked the `View Log` link, you will be redirected to the Jenkins instance that has been running your build.
-You might be prompted to log-in through Openshift, redirected to Openshit login screen and, if this is the first time you are using this account to
-login to Jenkins, asked to authorize information transfer between Openshift and Jenkins to facilitate the login.
+When the `View Log` link is clicked, you will be redirected to the Jenkins instance that has been running your build.
 
-===== Openshift login
+===== Openshift Login
 
-image::images/mobile-ci-cd-access-logs-3.png[jenkins-openshift-login]
+You might be prompted to log-in through Openshift and be redirected to the Openshit login screen to authorize information transfer between Openshift and Jenkins.
 
-image::images/mobile-ci-cd-access-logs-4.png[openshift-login]
+[.text-center]
+image::images/mobile-ci-cd-access-logs-1.png[jenkins-openshift-login]
 
 ===== Authorize access
 
-If you are asked to authorize permissions, you should be asked to approve access for:
+If this is the first time you are using the account to login to Jenkins, you will be asked to approve access for the following permissions:
+
 * user:info permission
-* user:authorize-access
+* user:check-access permission
 
-image::images/mobile-ci-cd-access-logs-5.png[authorize-access]
+[.text-center]
+image::images/mobile-ci-cd-access-logs-2.png[authorize-access]
 
 ==== Reviewing the build-log
 
-Now you should have been redirected to Jenkins UI, and see the log of the build. Based on your permissions to the Openshift project,
-you might have access to other Jenkins capabilities, such as inspecting the build configuration or re-running the build with changes in the pipeline script.
-
-image::images/mobile-ci-cd-access-logs-6.png[review-log]
+Now you should have been redirected to Jenkins UI, and see the log of the build. Based on your permissions to the Openshift project, you might have access to other Jenkins capabilities, such as inspecting the build configuration or re-running the build with changes in the pipeline script.
```

Comment:
Might read better as `Once you have been redirected to the Jenkins UI, you should see the log of the build.`